install-dev/upgrade/php/ps_1770_preset_tab_enabled.php

```diff
@@ -29,8 +29,8 @@
  */
 function ps_1770_preset_tab_enabled() {
     //First set all tabs enabled
-    Db::getInstance()->execute(
-        'UPDATE `'._DB_PREFIX_.'tab` SET `enabled`= 1'
+    $result = Db::getInstance()->execute(
+        'UPDATE `'._DB_PREFIX_.'tab` SET `enabled` = 1'
```

Comment:
```suggestion
        'UPDATE `'._DB_PREFIX_.'tab` SET `enabled` = TRUE'
```
Could we start to use MySQL boolean instead integer. Will be really easier to play with another database engine (on day maybe)